storybook/stories/contributing/contributing-get-ready.txt

```diff
@@ -0,0 +1,87 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contributing/Get Ready to Contribute" />
+
+Ready to start contributing to WPDS? There are a couple of steps you’ll need to take to get set up — we’ll take you through them.
+
+We understand GitHub might seem complicated if you’ve never used it before. Don’t worry — we’re happy to help you out with any questions in #design-systems on Slack.
+
+[Step 1: Get GitHub access](#step-1:-get-github-access)  
+[Step 2: Link your GitHub and WP.org accounts](#step-2:-link-your-github-and-wp.org-accounts)  
+[Step 3: Learn about the contribution process](#step-3:-learn-about-the-contribution-process)  
+[Step 4: Create your first PR](#step-4:-create-your-first-pr)
+
+#### **Step 1: Get GitHub access** {#step-1:-get-github-access}
+
+Our design system lives [on GitHub in the Gutenberg repository](https://github.com/WordPress/gutenberg) — it’s where we collaborate on the code and documentation, and where your contributions will be started, tracked, and finished.
+
+You’ll need a GitHub account to get started. If you don’t have one yet, [please sign up here](https://github.com/join).
+
+#### **Step 2: Link your GitHub and WP.org accounts** {#step-2:-link-your-github-and-wp.org-accounts}
+
+Then, you’ll need to connect your GitHub and WordPress.org accounts to make sure your contributions are correctly attributed. It’ll take just two minutes if you follow these steps:
+
+[Connect GitHub and WP.org accounts](https://make.wordpress.org/core/handbook/tutorials/linking-your-github-and-w-org-profiles/)
+
+#### **Step 3: Learn about the contribution process** {#step-3:-learn-about-the-contribution-process}
+
+In order to make effective contributions to WPDS, it’s important to know the basics of our contribution process. While we’ll take you through a step-by-step process depending on your contribution, we’ll explain the basics below.  
+   
+Depending on the contribution you’re making to the design system, you’ll either need to create an issue, or both an issue and a pull request (PR). 
+
+**What’s an issue?**
+
+Issues are how we keep track of changes to our design system on GitHub. You can find [all open issues on the Gutenberg repository](https://github.com/WordPress/gutenberg/issues).
+
+Smaller changes, like fixing a typo, don’t require you to create a PR yourself, and the implementation can be handled by the community. In this case, you can just create a new issue and add a clear description of the work you’re doing.
+
+Make sure to add the Design System label when creating an issue, and consider adding one of the following labels when needed:
```

Comment:
```suggestion
Make sure to add the [Design System ](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22Design%20System%22)label when creating an issue, and consider adding one of the following labels when needed:
```
I think it'd be handy to link the issues tab filtered by this label. We could do that as a standard for all mentions of labels.

Reply:
I like it! Adding and will apply to future writing.